File: InterconnectFrontend/src/models/enums/ConnectionStatus.ts
/**
 * Enumeration of possible connection statuses to the hypervisor.
 */
enum ConnectionStatus {
    /** Connection is active and alive */
    Alive = 1,
    /** Connection is dead or closed */
    Dead = 0,
    /** Connection status is unknown */
    Unknown = -1,
}

export { ConnectionStatus };

File: InterconnectBackend/Models/Enums/ConnectionStatus.cs
namespace Models.Enums
{
    /// <summary>
    /// Connection status with the hypervisor.
    /// </summary>
    public enum ConnectionStatus
    {
        /// <summary>
        /// Connection is active.
        /// </summary>
        ALIVE = 1,
        
        /// <summary>
        /// Connection is inactive.
        /// </summary>
        DEAD = 0
    }
}
